### DISCUSSION TITLE
Holy Unblocker - Main Discussion

### DISCUSSION THREAD

**Discussion by QuiteAFancyEmerald** [QuiteAFancyEmerald/InvisiProxy#40] [General]
### Community Discussion for Holy Unblocker
This mainly serves as a place where general discussions can take place. If you have any issues related to the official sites, please state them here or perhaps make an issue. You may also leave any suggestions you have here.

However, I strongly advise that you join the Titanium Network discord for anything feedback-related or better support.

### General FAQ
**What is Holy Unblocker?**
Holy Unblocker is a secure web proxy service supporting numerous sites while concentrating on detail with design, mechanics, and features. Bypass web filters regardless of whether it is an extension or network-based!

**What is Titanium Network?**
TitaniumNetwork is an organization dedicated to providing private internet access to information by bypassing over-restrictive filters.

**What proxy is best for YouTube support?**
Scramjet has the best support since it allows viewing comments, streams, and full video quality. 

**Regarding the proxies on Holy Unblocker, what sites are supported?**
The below is ranked in order of most to least support.

Join the TN discord and view #services for more information.

**How do I obtain more Holy Unblocker sites that are not blocked by my school district or administration?**
If you join the TN discord, you can click on the button in #panel and the bot in the server will DM you another site. Sites are restocked on a weekly basis and you are limited to 3-9 sites per month, depending on if you are boosting the server or donating.

TN Discord: https://discord.gg/unblock

**More information:**

**If you follow any of the criteria above, sorry. I can't do anything about it. Please do not ask for more sites or spam issues.**

**Exception:** "The sites I got from the proxy bot were blocked for my filter." Open a ticket in the discord about this. You sometimes obtain sites from the proxy bot meant for other filters. By sending me a screenshot (proof of it being blocked) I'll provide you with another Holy Unblocker site for your particular filter.  

Lastly (sad that I need to define this) a filter is what your district, school, or workplace uses to block sites. People often call them "blockers" here.

#### ============ Epic Discussions Happen Below :D  ============

**Comment by D4rkC0met** [comment]
@QuiteAFancyEmerald what are the alternative links, I'm using it to get on discord and it's the only way I can access discord so I can't check those alt links

**Reply by QuiteAFancyEmerald** [reply]
Alright, I emailed you a site and deleted your original message for some privacy.

**Reply by D4rkC0met** [reply]
okay, thank you very much!

**Reply by Susdevlmao** [reply]
Emerald Ur amazing im just suggesting it but maybe another form of people getting links for holy unblocker maybe u code a bot that sends it to ur email or maybe uh a gc or maybe a private repository on github for people who are active the most on gihub just an idea

**Reply by QuiteAFancyEmerald** [reply]
Hmm no both of those ideas are not really good. Why not just join the Discord and request for sites through the proxy bot? If you don't have another device, you are out of luck sorry.

Also once again, please quit asking for more Holy Unblocker links. Just wait until I post them on the review pages, to those who are asking. :(

**Reply by russianjesus52** [reply]
i just steal my sisters tablet and get onto her discord for the links lol

**Reply by Susdevlmao** [reply]
lmao

**Reply by maiku0** [reply]
i wait til no1 is home so i can sneak on discord for the links :/

**Reply by idk56789098765** [reply]
> Alright, I emailed you a site and deleted your original message for some privacy.

im not sure what to do , holyunblockers isnt working for discord and the links im being told to use arnt working either what are some things i can use to get on discord

**Comment by masterful21** [comment]
So Emerald are you able to manage all of the code? cuz I don't know if this is just my slow Chromebook but on cookie clicker the sugar lumps when I try to apply one to a building the game freezes. Also all of the upgrade and achievement icons don't appear.

**Reply by QuiteAFancyEmerald** [reply]
I'm not sure. I don't actually manage Cookie Clicker itself of course but I'll look into it.

**Reply by ghost** [reply]
I was playing and had a similar issue. I think it could just be because we are using a proxy but idk

**Comment by MrMachi9** [comment]
can you email me a krunker unblocked site

**Reply by QuiteAFancyEmerald** [reply]
Not a thing sorry

**Comment by 3797133146** [comment]
holy unblocker is blocked

**Reply by QuiteAFancyEmerald** [reply]
#44 sorry

**Reply by nigqers** [reply]
only environmental works for me

**Reply by JoeyMeepLand** [reply]
all of those are blocked

**Reply by cqnce11ed** [reply]
none works for me dns error

**Comment by ghost** [comment]
Quite A Fancy Emerald I would like to say great job on this site. You and your team did an awesome job at getting this thing going. I love the games and the wide support for sites this provides. You guys keep up the great work!

**Comment by Brady-Shike** [comment]
Can you email me an alt site

**Reply by Susdevlmao** [reply]
#44

**Comment by Susdevlmao** [comment]
Suggestion : Pluto.tv or Tubi.tv?????

**Reply by Brady-Shike** [reply]
Pluto

**Reply by Susdevlmao** [reply]
YES

**Comment by cooperllama** [comment]
You should add candy box 2 to the html5 games list

**Comment by russianjesus52** [comment]
@QuiteAFancyEmerald my friend wanted to know, will there ever be a mobile version?

**Reply by Susdevlmao** [reply]
there used to be i think if not then they are making it lemme check

**Reply by russianjesus52** [reply]
interesting

**Comment by Susdevlmao** [comment]
Day 1 of asking @QuiteAFancyEmerald to support pluto.tv or tubi.tv 
FOR THE MEME 
THE DREAM

**Comment by Eman10826** [comment]
Hey Holy Unblocker has been blocked for me how do I unblock it

**Reply by QuiteAFancyEmerald** [reply]
Read over #44 regarding this :D

**Reply by maiku0** [reply]
what if i log in discord and it says there seems to be some werid activvity then what?

**Reply by QuiteAFancyEmerald** [reply]
In that case you would need another device or be able to access your email. You could also try again later. :(

Unfortunately there isn't much you can do about it. (Note that this issue would be universal regardless of what you would be using to access discord eg. web proxy, vpn, etc.)

**Comment by maiku0** [comment]
@QuiteAFancyEmerald  when i try to login on my discord this pops up 
![tempsnip](https://user-images.githubusercontent.com/87177320/125153000-b1dbf300-e105-11eb-9fe7-805c0a78a2ca.png)

**Reply by maiku0** [reply]
any help?

**Reply by foxsouns** [reply]
@maiku0 it means that so many people are using the same site as you for discord, so discord has flagged that ip, as to discord it looks like youre a bot. you could try using another link, or host your own proxy.

**Comment by maiku0** [comment]
thanks

**Reply by Susdevlmao** [reply]
lol thats not the proxys fault u need to actually verify it 
if u cant get a phone then sorry i tried to bypass it
discord a tough not to crack (lol)
dont try to bypass 
get a phone

**Comment by maiku0** [comment]
epik
![tempsnip](https://user-images.githubusercontent.com/87177320/125184578-c71f5300-e1d3-11eb-801e-b7cb92e5ffb1.png)

**Reply by maiku0** [reply]
only 2 away :(

**Comment by maiku0** [comment]
anyone tryna browse???

https://tutturu.tv/i/K9wUlEVJ

**Reply by JoeyMeepLand** [reply]
blocked

**Comment by maiku0** [comment]
perfection
![SharedScreenshot](https://user-images.githubusercontent.com/87177320/126083232-c39ed33d-5b16-4fa7-9dd3-f668ba9db4bc.jpg)

**Comment by maiku0** [comment]
@QuiteAFancyEmerald 

sorry for pinging u but do u know any other mirror links?

**Reply by 6829ASCS** [reply]
#44 

We can't send mirror links here since Securely will easily find them here on GitHub and block them instantly... Leaking links is the whole reason Holy Unblocker is down right now:/

**Reply by maiku0** [reply]
oh is that why nothing works on them?

**Reply by maiku0** [reply]
oh i dont have securely on my laptop
 
i have another school one not popularly used one

**Comment by martyhil123** [comment]
@QuiteAFancyEmerald  hey, it's me sp00ked#2264
I saw that I was kicked from the tn discord server and unfriended by you.

I just want to say, I apologize for any trouble I might have caused you or the team. But, I have no relation with the SCP wolf discord server, I was banned there along time ago.
SCP is a guy i'm not very fond of, I asked for his assistance if I could have some proxies / methods to unblocking sites and he basically turned me away and made me feel like shit.
I don't want to drag the story but i'll sum it up.
I asked for his phone number because I myself don't have one (as you know) for one of his site methods. BAD IDEA. The idea I had in mind was very different from his and he really pushed himself on me. I apologized then and continued to ask him for alternate methods. He ignored me and continued to until I asked for that specific method. (keep in mind this was in dms)
He told me it was wrong to ask him for his number and gave me a choice; leave his server or actually receive help. it's obvious which one I picked. At the time I was still in the server I saw a ping notification,  #general to be exact. I clicked it. To my surprise, SCP @everyone'd in the server about our conversation. I was furious.
He kicked me from his server for 'scamming' and 'being a karen' and blocked me.
I joined the server and was kicked about 3 times until I was banned. I made an alternate account and joined the server. I then saw his message 
"NO"
"I have no obligation to help you"
I left it at that. 
If you don't believe me, you can go to his server and scroll until you find the 'goodbye' channel and search for my user 'Sp00ked'. You can see I never joined the server again after those 3-4 times.
I know you guys are really cracking down on alternate accounts and the like, but i'm being serious when I say I don't have any relation with the server. Even when I was in the server, I never gave them methods, especially not from the tn bot.

When I was kicked after 1-2 times away from when I was in a state of statis and decided to lay low I messaged Dannyboi for some methods (at this time he said he boasted having a plethora of methods and wanted to be a mod or a contributor) I messaged him, he said something along the lines of 'ok' or 'alright' but never got back to me.

That's the truth, really. I promise you i'm not a bot. Fancy, you're a really nice guy. I just wanted to clear my name. thank you.

**Reply by QuiteAFancyEmerald** [reply]
Are you asking to be unbanned or be able to change accounts in the server? Very confused lol
I brought this up to the staff team but it was concluded that you were never banned but merely lost your main account?

**Reply by martyhil123** [reply]
I want you to friend me again and I want to get back in the server
I'm not sure if the server is on lockdown, the vanity didn't work

**Reply by martyhil123** [reply]
Did you unfriend or block me?
I swear I have nothing to do with scp, i don't like the guy one bit

**Reply by QuiteAFancyEmerald** [reply]
I didn't do anything yet wha
Also you never answered my question

**Reply by martyhil123** [reply]
Sorry about that.
I saw the bot notification and it only said i was kicked (not banned or anything) but I can't join the server. I don't have any alts and wil is my brother (you can see we talk really differently)
Well if I'm banned then yeah, I want to be unbanned

**Reply by QuiteAFancyEmerald** [reply]
You don't seem to be banned. Make sure you are using the updated invite (which you can find on HU). Also make sure you are not trying to join on a proxy or anything.

**Reply by martyhil123** [reply]
@QuiteAFancyEmerald My brother sent me an invite. I joined.  But why did you unfriend me? I saw i got kicked and unfriended by you so i instantly thought it was related to SCP.

**Comment by Tylerodi** [comment]
Suggestion: tinder also twitter and maybe Instagram please

**Reply by D4rkC0met** [reply]
tinder?

**Comment by maiku0** [comment]
@QuiteAFancyEmerald sorry for asking and pinging you but titanium network wont work for me says its blocked any help?

**Comment by YEEHAW007** [comment]
Maybe add a Capcom CPS1/CPS2/CPS3 emulator so you can play Street Fighter 2 or Final Fight. For the Retroarch core I suggest Final Burn Neo.

**Reply by foxsouns** [reply]
you could, in theory, fork https://github.com/BinBashBanana/webretro and add some more cores onto it, and just throw it onto replit and selfhost. i dont think there should be any problems, considering how well the other oldgen consoles work on webretro.

**Comment by hurstbergn** [comment]
hey uh, is there a way i can get unbanned from the titaniumnetwork discord? i got banned a couple months ago, and i really need to get another link.
user is glove#8228

**Comment by BannableBalls** [comment]
****gives me a 403 unknown error wizards curtains when i try using holyunblocker****

**Reply by foxsouns** [reply]
send a screenshot

**Reply by isaiahcap** [reply]
dose someone have a YouTube site?

**Comment by D4rkC0met** [comment]
@QuiteAFancyEmerald the womginx stealth proxy for discord bugged out so its not loading anything and i logged out and tried logging back in but the login button loads for half a second then does nothing. no its not a slow internet problem and ive tried other links, it doesnt say incorrect password/email either it just does literally nothing

any idea as to why?

**Reply by D4rkC0met** [reply]
nevermind its fixed sorry

**Comment by 3amonEamon** [comment]
so i dont have a phone and my discord account got flagged for the ip thing and its the thanksgiving break. 6 more days until i get to school and ask my friend for his phone. also most of my links have been blocked due to relay lightspeed. no more links :(

**Comment by SuperiorSpiderman** [comment]
@QuiteAFancyEmerald I am using the Holy Unblocker on Heroku, which I am extremely new to, and I have little to no coding experience. It says application error whenever I deploy the master branch and view the app. Any ideas on how to fix this?

**Reply by ghost** [reply]
Speaking of Heroku, @QuiteAFancyEmerald  HOW DO YOU INSTALL THE PROXYS ON IT???
I am quite new to this whole proxy thing, and I am on a school Chromebook, trying to get to discord.
So can you like walk me through the process?

**Reply by ghost** [reply]
Nevermind
I just figured it out 😂

**Comment by ihateschool1** [comment]
@QuiteAFancyEmerald. I #44 it says to provide proof of bloackage by you current school.
![Screenshot 2021-12-14 12 51 47 PM](https://user-images.githubusercontent.com/96145254/146069642-c977cc3d-4f73-4bb4-a73a-5b5b5938fc8c.png)

**Comment by JoeyMeepLand** [comment]
when is the sever coming back up

**Comment by keevan163** [comment]
Same

**Comment by mistertoenails** [comment]
Hey em, I've tried to reach you on reddit because my holy unblocker link is broken so i cant get discord, and tried to message with an alt url for old reddit but I'm not sure if it went through. I want to try to code my own proxy, not just forking or designing yet another corrosion frontend, and would love to learn from the best. I can send you a repl link if you're interested, hope to hear from you!

**Comment by KaiiUvU** [comment]
does anyone have a discord unblocker
![Screenshot 2022-04-18 11 48 01 AM](https://user-images.githubusercontent.com/103742601/163834603-9710febe-8eea-4046-b315-799b6848815f.png)
?

**Comment by Epic-gamer6969** [comment]
When i host the site on replit.com the games and emulators dont work. Is their a potential fix?

**Comment by Blooketplayer99** [comment]
Is there another way to use this because my district blocked the website?

**Reply by QuiteAFancyEmerald** [reply]
Join the discord at discord.gg/unblock; request for another domain through the panel. You can also self host this

**Reply by Blooketplayer99** [reply]
Discords blocked and I have no other way to access it

**Reply by QuiteAFancyEmerald** [reply]
Fork this repo and try to use GitHub Codespaces or Gitpod; then use that to access the discord
